src/distributions/uniform.rs

```diff
@@ -884,11 +898,19 @@ pub struct UniformFloat<X> {
 }
 
 macro_rules! uniform_float_impl {
-    ($ty:ty, $uty:ident, $f_scalar:ident, $u_scalar:ident, $bits_to_discard:expr) => {
+    ($($meta:meta, $doc:literal)?, $ty:ty, $uty:ident, $f_scalar:ident, $u_scalar:ident, $bits_to_discard:expr) => {
+        $(#[doc = $doc]
+        #[cfg(feature = "simd_support")]
+        // TODO: as doc_cfg/doc_auto_cfg mature ensure they catch this
+        #[cfg_attr(doc_cfg, doc(cfg(feature = "simd_support")))])?
         impl SampleUniform for $ty {
             type Sampler = UniformFloat<$ty>;
         }
 
+        $(#[doc = $doc]
+        #[cfg(feature = "simd_support")]
+        // TODO: as doc_cfg/doc_auto_cfg mature ensure they catch this
+        #[cfg_attr(doc_cfg, doc(cfg(feature = "simd_support")))])?
```

Comment:
Same here... (`UniformFloat` docs)
![image](https://github.com/rust-random/rand/assets/134893/2891665a-701e-49a1-883e-8813815697ca)
... but not here (trait `UniformSampler` omits docs):
![image](https://github.com/rust-random/rand/assets/134893/79760358-ede4-4d8f-8816-fa293df0faa8)